src/patient-registration/input/dummy-data/dummy-data-input.component.tsx

```diff
@@ -1,9 +1,10 @@
+import { useFormikContext } from 'formik';
 import React from 'react';
 import { FormValues } from './../../patient-registration.component';
 import styles from './../input.css';
 
 interface DummyDataInputProps {
-  setValues(values: FormValues, shouldValidate?: boolean): void;
+  setValues?: { (values: FormValues, shouldValidate?: boolean): void };
```

Comment:
No need for the ceremony here. Just insert the `?`, but keep the rest as is:

```ts
setValues?(values: FormValues, shouldValidate?: boolean): void;
```

Reply:
The reason for the ceremony actually was a parse error `Parsing error: Unexpected token, expected ":"`(by eslint from what I understand). Same for the occurrence in `demographics-section.component.tsx`

Reply:
Hm it should not complain. Maybe its outdated?

If the tooling does not understand valid language constructs then the tooling has to change ;). I can have a look if it still appears to be broken with the tooling.

Reply:
I upgraded `eslint` to the lastest version, but am still getting the same error. I'm not familiar with configuring eslint for react + typescript, but upgrading `eslint-config-ts-react-important-stuff` to the latest version results in more linter errors (`Parsing error: The keyword 'interface' is reserved`), so I did not commit it. 
I'm happy to hear any advice, but feel free to also have a look yourself, if that's easier to move forward with the PR.